swap/config.go

```diff
@@ -16,14 +16,16 @@
 
 package swap
 
-import "time"
+import (
+	"time"
+)
 
 // These are currently arbitrary values which have not been verified nor tested
 // Need experimentation to arrive to values which make sense
 const (
 	// Thresholds which trigger payment or disconnection. The unit is in honey (internal accounting unit)
-	DefaultPaymentThreshold    = 1000000
-	DefaultDisconnectThreshold = 1500000
+	DefaultPaymentThreshold    = 2441 * (RetrieveRequestPrice + ChunkDeliveryPrice) // Payment when the disbalance is an equivalent of 10 mb of data (2441 chunks)
```

Comment:
RetrieveRequest and ChunkDelivery are added because I wanted the 
DefaultPaymentThreshold to be equal to 10mb worth of Honey (10 mb = 2441 chunks).

I see your comment is correct and that 10mb worth of honey is not `2441 * (RetrieveRequestPrice + ChunkDeliveryPrice)`, but `10^6 * ChunkDeliveryPrice + 2441 * chunkRequestPrice`.

Reply:
each chunk is `4096` bytes, right? so `4096*2441` is ~`10` megabytes. is that where `2441` comes from?

where does `10^6` come from though? should it not be `10^7`?

Reply:
Thanks @mortelli , 10^6 is bytes to megabytes, but this is 10 megabytes, so 10^7.

Reply:
I added a comment, explaining where the numbers come from